packages/core/src/components/portal/portal.md

```diff
@@ -30,4 +30,18 @@ application.
 `Portal` supports the following options on its [React context](https://facebook.github.io/react/docs/context.html).
 To use them, supply a child context to a subtree that contains the Portals you want to customize.
 
+<div class="@ns-callout @ns-intent-primary @ns-icon-info-sign">
+    Blueprint uses the React 15-compatible `getChildContext()` API instead of the newer React 16.3 `React.createContext()` API.
```

Comment:
What if you want to use `React.createContext()` or some other new React feature in the future? Supporting both versions might seem like a few hundred LOC right now, but it could end up being pretty limiting.

Reply:
i agree, it is quite limiting, but it's the quickest path to adoption. shim/ponyfill packages are available, such as [create-react-context](https://github.com/jamiebuilds/create-react-context), which is actually used in the latest version of react-popper to support multiple versions. so we could go down that route, like we did with pure-render-decorator.